enterprise-cfengine-guide/settings.markdown

```diff
@@ -66,12 +66,25 @@ will not see any hosts in Mission Portal.
 Users without a role will not be able to see any hosts in Mission
 Portal.
 
-**Default Roles:**
+**Predefined Roles:**
 
 * admin - The admin role can see everything and do anything.
 * cf_remoteagent - This role allows execution of cf-runagent. It can
   be used from within Design Center to troubleshoot hosts that have
   failed sketch activations.
+  
+**Default Role:**
+
+To set a default role navigate to the Role Management page and checking the boxes in the Default role column which you would like to apply to all new users.
+
+![DefaultRoleSelecting](roles-list.png)
+
+**Behaviour of Default Role:**
+
+Once you set up default role, all new users registered through LDAP would have this role.
+When you add new user, the default role will be already selected.
```

Comment:
Suggesting:
Any new users created in Mission Portal's local user database will have this new role assigned.

Users authenticating through LDAP (if you have LDAP configured in Mission Portal) will get this new role applied the first time they log in.

Note that the default role will not have any effect on users that already exist (in Mission Portal's local database) or have already logged in (when using LDAP).

Reply:
In effect this allows you to set the default permissions for new users (e.g. which hosts a user is allowed to see) by configuring the access for the default role.

Reply:
I've added your suggestions, thanks!